test/snippet/io/sequence_file/sequence_file_input.cpp

```diff
@@ -0,0 +1,216 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/input.hpp>
+#include <seqan3/range/container/bitcompressed_vector.hpp>
+#include <seqan3/range/view/get.hpp>
+#include <seqan3/std/view/filter.hpp>
+
+using namespace seqan3;
+
+struct my_traits : sequence_file_input_default_traits_dna
```

Comment:
No, this is required here because the snippet in line 31 [trait_overwrite] defines the struct, but it can't be defined within `main()`. But I realized that I didn't actually test the trait overwrite, so I will add that below the `#endif`.